CoCoding/apps/codes/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CodesConfig(AppConfig):
+    name = 'codes'
```

Comment:
dude! you wanna die? have you decided to ruin our project????right?

Reply:
huk...I'm sorry my bad